cocotb/share/lib/fli/FliObjHdl.cpp

```diff
@@ -341,7 +347,7 @@ int FliLogicObjHdl::set_signal_value(std::string &value)
             mti_SetSignalValue(get_handle<mtiSignalIdT>(), enumVal);
         }
     } else {
-        
```

Comment:
Mind doing these whitespace cleanups in a standalone PR? They make reviewing a little awkward

Reply:
There is a radio box in the top right corner when reviewing to turn off whitespace diffs. You can add `?w=1` to the URL if you can't find it. My editor auto-trims trailing whitespace, I don't do them manually. I will do a whitespace cleanup PR in future, but we really should run whitespace checks on PRs, I have no clue how they get in there anyways, are people still using WordPerfect?